code/web/interface/themes/responsive/css-rtl/results-list.less

```diff
@@ -20,6 +20,11 @@
   font-weight: bold;
   color: #44484a;
 }
+.fa-angle-down {
```

Comment:
I'm surprised there's a need for this.. do we now user other font awesome assets in a similar way? I don't see targetted styles for those cases?

Less code is easier to maintain code ;P